### PR TITLE
Bme680

### PR DESCRIPTION
**Pull request by mcgrailk3**
Need to figure out why it crashes when doing the i2c pin assignment from the menuconfig
Also need to add ability in menuconfig to enable/disable each sensor instead of just commenting out

### CODE REVIEW

File: main/Kconfig.projbuild [hunk 1]
```diff
@@ -68,11 +68,7 @@ menu "Monster Configuration"
             Minimum CPU frequency to use for dynamic frequency scaling.
             Should be set to XTAL frequency or XTAL frequency divided by integer.
 
-        config EXAMPLE_MIN_CPU_FREQ_40M
-            bool "40 MHz (use with 40MHz XTAL)"
-            depends on IDF_TARGET_ESP32S2 || ESP32_XTAL_FREQ_40 || ESP32_XTAL_FREQ_AUTO
-        config EXAMPLE_MIN_CPU_FREQ_20M
-            bool "20 MHz (use with 40MHz XTAL)"
+        config EXAMPLE_MIN_CPU_FREQ_40M  #endif)"
```
Comment (Kanaderu) [inline main/Kconfig.projbuild:71]: Is this change intentional? I think this is likely incorrect

File: partitions.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# ESP-IDF Partition Table
```
Comment (Kanaderu) [inline partitions.csv:1]: May want to add the partitions.csv setup as default in the sdkconfig.default setup, otherwise I'd probably remove this

Comment (Kanaderu) [reply partitions.csv:1]: added default partitions with 7d0ca95

File: main/Kconfig.projbuild (hunk 1 above)
Comment (mcgrailk3) [reply main/Kconfig.projbuild:71]: Yeah I have no clue why that changed, I did not change it

**Comment by Kanaderu** [comment]
I think it'll be good to add in the ability to enable/disable sensors with the menuconfig before merging it with master.

File: CMakeLists.txt [hunk 3]
```diff
@@ -2,7 +2,8 @@
 # in this exact order for cmake to work correctly
 cmake_minimum_required(VERSION 3.5)
 
-set(EXTRA_COMPONENT_DIRS $ENV{IDF_LIB_PATH}/components)
+#set(EXTRA_COMPONENT_DIRS $ENV{IDF_LIB_PATH}/components)
+set(EXTRA_COMPONENT_DIRS /home/viper/esp/esp-idf-lib/components)
```
Comment (Kanaderu) [inline CMakeLists.txt]: CI test failing here probably
